teos/src/config.rs

```diff
@@ -1,6 +1,7 @@
 //! Logic related to the tower configuration and command line parameter parsing.
 
 use serde::Deserialize;
+use serde::Serialize;
```

Comment:
Let's combine these two imports.

Reply:
This hasn't been resolved

Reply:
> That and not resolved is the same with respect to reviewers. Push the changes you are sure about and ask about the others, then you can push again once the conversation has been resolved.
> 
> (Or at least do not resolve something on GitHub if you have only addressed it locally)

Okay got it....I have pushed the latest changes. Should I resolve the conversation now or after reviewing?